samples/package.json

```diff
@@ -14,6 +14,8 @@
   "dependencies": {
     "@google-cloud/speech": "2.0.0",
     "@google-cloud/storage": "^1.7.0",
+    "mime": "^2.3.1",
```

Comment:
What're these packages for? Are they being used somewhere in the code?

Reply:
More than a few modules use it as dependency- we had this before, I was just trying to account for latest update - https://www.npmjs.com/package/mime